src/components/cluster/Cluster.tsx

```diff
@@ -346,6 +352,13 @@ function Cluster({
                 required: true,
                 validator: { error: 'URL is required', regex: /^.*$/ },
             },
+            proxyUrl: {
+                required: false,
+                validator: { error: 'Incorrect Url', regex: /^.*$/ },
+            },
+            toConnectViaProxy: {
```

Comment:
Is this needed?

Reply:
if this is not present, handleOnChange validator gives error as it is not able to get the key present in the object. we do not need the validation but we need the key to be present for the validator logic to not break